qutebrowser/browser/hints.py

```diff
@@ -748,6 +760,13 @@ def start(self, rapid=False, group=webelem.Group.all, target=Target.normal,
         self._context.tab.find_all_elements(selector, self._start_cb,
                                             only_visible=True)
 
+    def current_mode(self):
+        """ Returns the currently active hinting mode (or None otherwise). """
```

Comment:
I removed the spaces inside the docstring here - now that one is a common [python convention](https://www.python.org/dev/peps/pep-0257/#one-line-docstrings).